README.md

```diff
@@ -10,8 +10,9 @@
 
 *The graph database for delightful React state management*
 
+State management that's simple to code and maintain in just 3 lines of code.
```

Comment:
This line is confusing me. Do we need it? What are you trying to accomplish with it that's different from the line above?